README.md

```diff
@@ -1,5 +1,13 @@
 # VGA Simulator
 
+**Note:** This project is not actively maintained. Consider using one of the following newer ports:
+
+- [Pedro, @pvieito](https://github.com/pvieito) made a CLI version of this app in Python that you can find in [this gist](https://gist.github.com/pvieito/8cdb54a9a03fd36e51c8df6e331a3006).
```

Comment:
It looks like @pvieito also made a Swift port we can link here too: https://github.com/pvieito/VGASimulatorKit